worlds/sc2/docs/custom_mission_orders_en.md

```diff
@@ -843,6 +853,99 @@ Grid supports the following index functions:
 ##### rect(x, y, width, height)
 `rect(x, y, width, height)` returns the indices within the rectangle defined by the starting point at the X and Y coordinates and the width and height arguments. In the above example, `rect(1, 2, 3, 2)` returns the indices `11, 12, 13, 16, 17, 18`.
 
+---
+### Canvas
+```yaml
+type: canvas
+canvas: # No default
+jump_distance_orthogonal: 1 # Accepts numbers >= 1
+jump_distance_diagonal: 1 # Accepts numbers >= 0
+```
+
+This is a special type of grid that is created from a drawn canvas. For this type of layout `size` may be omitted, but `canvas` is required.
```

Comment:
Checking the code, `size` is ignored/overwritten if specified; this doc isn't clear about that. Maybe "For this type of layout, `canvas` is required; `size` is ignored if specified, and determined from the `canvas` dimensions."

Reply:
The immediate next sentence says how size is determined, so I'll leave out the last part.